src/main/java/seedu/address/logic/parser/PoolCommandParser.java

```diff
@@ -22,10 +22,14 @@
 import seedu.address.model.tag.Tag;
 
 public class PoolCommandParser implements Parser<PoolCommand> {
+
     /**
      * Parses the given {@code String} of arguments in the context of the PoolCommand
      * and returns an PoolCommand object for execution.
-     * @throws ParseException if the user input does not conform the expected format
+     *
+     * @param args The arguments to be parsed.
```

Comment:
would it be better without `the` since the rest of your documents don't include `the`

Reply:
good catch, thanks. Changes will be made